### ISSUE TITLE
UndefVarError: ColorProcessor not defined

### ISSUE THREAD

**Issue by ederag** [cesaraustralia/DynamicGridsGtk.jl#22]
The example fails:
```julia
using DynamicGridsGtk
output = GtkOutput(init; tspan=1:100, fps=25, showfps=fps, store=false, processor=ColorProcessor())
ERROR: UndefVarError: ColorProcessor not defined
```
`using DynamicGrids` does not help. The Readme says

> processor can be any `GridProcessor` from DynamicGrids.jl

But `GridProcessor` is found only once, in the [DynamicGrids doc](https://cesaraustralia.github.io/DynamicGrids.jl/dev/#Output-interface). 

_Edit: same issue with [DynamicGridsInteract](https://github.com/cesaraustralia/DynamicGridsInteract.jl#dynamicgridsinteract), 
and a search on "ColorProcessor" yields almost nothing,
so it does not seem to come from another package. Looks like the API changed ?_

**Comment by rafaqz** [comment]
Oh yeah sorry this is old. I will fix. Thanks for finding and reporting these. 

Its the problem with putting examples in readmes, where they dont get tested.

Im also working on adding a fee more conplex examples currently. They should be pushed to the DynamicGrids.jl docs in a few days.

**Comment by rafaqz** [comment]
For now just delete that argument, its all automatic now. `show_fps` is also old.

The actual doc strings should be correct. Check them in the REPL

**Comment by ederag** [comment]
Thank you for DynamicGrids, awesome work !

And indeed, with
```julia
output = GtkOutput(init; tspan=1:100, fps=25, store=false)
```
the Gtk window appears fine.


The REPL help still shows `processor=ColorProcessor()` in the top lines
```
  GtkOutput(init::AbstractMatrix; fps=25, store=false,
            processor=ColorProcessor(), extrainit=Dict())
```
but the `Keywords` list is correct.

Pointing to this list might be useful, as the `maxval` does not seem automatic
(looks like it is set to 1, rather than updated upon each frame when missing).
Another place where a full list would be useful is in
https://cesaraustralia.github.io/DynamicGrids.jl/stable/#DynamicGrids.GraphicConfig

**Comment by rafaqz** [comment]
`maxval` cant really be automatic, or it will move during the simulation. 

You really _don't_ want that... the values of some cells will affect the colors of other cells, giving you a false impression of the dynamics.

Its just like when you have to set `clims` to do comparative plotting of multiple heatmaps of the same variable. We basically have hundreds of comparative plots.

I really should call it `clims` so that comparison is clearer to people.

**Comment by rafaqz** [comment]
Those `GraphicConfig` args should aslo be in the docstring here - they are passed on. I really need to do some more work on getting this all right.

**Comment by ederag** [comment]
Sure, in the end one usually wants to fix the scale
(except perhaps when the relative morphology is more important than the absolute values).

But the first time trying DynamicGrids on a new problem, it could make sense so have an autoscale,
just to see that the simulation is running properly.
That's how I incorrectly interpreted "all automatic now" above.

With the current setting, if the cells are all above 1, the image stays white (except happily the frame number),
but with the proper documentation of `maxval` (or `clims` which would be fine indeed),
it would be very easy for the user to diagnose and fix that, agreed.

**Comment by rafaqz** [comment]
Ok this makes sense. Do you want to open a new issue about maxval?
